project/package/CodeEditor.vue

```diff
@@ -386,6 +390,21 @@ export default {
   margin: 0;
 }
 
+count-lines{
+  counter-reset: line;
+}
+.count-lines:deep(span.hljs-tag){
+  counter-increment: line;
+}
```

Comment:
Change line 399 to .count-lines:deep(span:before){
So that every span gets a line number, regardless of the class highlight js gives.

Reply:
This can't work because inside span.hljs-tag the editor can add sub-span with other classes, this would increment incorrectly the line-number.

Look the screenshot

![Schermata 2022-10-18 alle 11 32 00](https://user-images.githubusercontent.com/36926081/196393621-100794b6-294a-45f3-a76f-f08329673da8.png)

Reply:
Applied 👍🏻